tools/Dashboard/DevHome.Dashboard/Controls/WidgetControl.xaml.cs

```diff
@@ -177,7 +177,10 @@ private async void OnRemoveWidgetClick(object sender, RoutedEventArgs e)
                 _log.Debug($"User removed widget, delete widget {widgetIdToDelete}");
                 var stringResource = new StringResource("DevHome.Dashboard.pri", "DevHome.Dashboard/Resources");
                 Application.Current.GetService<IScreenReaderService>().Announce(stringResource.GetLocalized("WidgetRemoved"));
-                Application.Current.GetService<DashboardViewModel>().PinnedWidgets.Remove(widgetViewModel);
+                await Application.Current.GetService<DispatcherQueue>().EnqueueAsync(() =>

```

Comment:
Does the `DispatcherQueue` only have an async enqueue?

Reply:
Whoops, no. Will fix.